### ISSUE TITLE
Filters are not working perfectly

### ISSUE THREAD

**Issue by gauravpansari1991** [Sitecore/XM-Cloud-Introduction#425]
**Describe the bug**
When I select 2024 and the United Arab Emirates facet then it is also showing the Jordan facet and its 1 result along with UAE profile cards

**To Reproduce**
1. Go to https://mvp.sitecore.com/en/Directory/
2. Select '2024' in year filter
3. Select 'United Arab Emirates' in country filter
4. It should only show profiles of UAE but it is also showing 1 Jordan entry

<img width="340" alt="image" src="https://github.com/Sitecore/XM-Cloud-Introduction/assets/12952868/ba18ad2c-bc17-485f-adfe-9dce683841b5">

**Expected behavior**
It should only show 10 results of the United Arab Emirates.

**Comment by sc-ivanlieckens** [comment]
@gauravpansari1991 this result is actually correct due to this particular person having been MVP in multiple countries:
![image](https://github.com/Sitecore/XM-Cloud-Introduction/assets/72252314/dddda9c8-2487-4e19-ae35-fb1ba5dc8788)

@mrtamasvarga would you want this altered so it only takes "current" country into account or do we leave it to surface all countries of the MVP?

**Comment by gauravpansari1991** [comment]
@sc-ivanlieckens makes sense but looks a little bit weird in the UI. Also, this is working fine for my profile though, as I was MVP in India for 3 years and then moved to Dubai, but my profile is not showing in India.

**Comment by sc-ivanlieckens** [comment]
@gauravpansari1991 well when I look at your titles none of them are in India? So logical that yours isn't showing in India.
![image](https://github.com/Sitecore/XM-Cloud-Introduction/assets/72252314/688ff102-848d-4ade-8e24-14f11fe93d52)

I can correct your title data to fix this. What years should I change to India?

I'll need to check the facet counting because it seems that results can't be counted twice which does give a slight miscount there.

**Comment by gauravpansari1991** [comment]
@sc-ivanlieckens 2017, 2021, 2022 should be tagged to India.

**Comment by IvanLieckens** [comment]
@gauravpansari1991 I have updated the records so that should resolve this issue

**Comment by robearlam** [comment]
@IvanLieckens - Having a think about this, I agree it seems strange to have what looks like an incorrect country included in the results, even if the data is correct.

Instead of listing all applicable countries on the results listing page, which could get messy fast. What do you think about changing it say "Multiple" for MVP profiles who have awards from different countries?

**Comment by sc-ivanlieckens** [comment]
@robearlam or maybe do a flyout with countries on hover or something?

**Comment by peplau** [comment]
When I found this one, I was about to file a new bug and thought they might be related.

The attached image shows the filters applied: 
- Year: 2025
- Type: Technology
![Image](https://github.com/user-attachments/assets/fde38afc-9b44-4425-87a2-2e8377bfdb4f)

Still, some entries are showing MVPs from other categories. 

Is this happening because those were Technology MVPs in past years? If that's the case, is the filter applied with the OR logical operator? 

I'm not sure about other users, but when I applied the filters, I was trying to list all MVPs in 2025, regardless of their statuses in past years. If I select 2025 and 2024, I would expect to see only the MVPs in both 2024 AND 2025 years. Thus, it would be an AND operator.

**Comment by robearlam** [comment]
@mrtamasvarga - How do you want to handle users who have awards in different categories, different countries. 

Currently the Profile card only lists the "latest" value for each, which leads to the confusion like this. 

It was suggested above for those users to change the Profile Card to state "multiple" for those fields and have a hover fly out that would list all of the Awards/Countries that have. Would you be happy with that approach?
